pennylane/ops/op_math/condition.py

```diff
@@ -581,8 +583,8 @@ def qnode(a, x, y, z):
 
     if elifs:
         raise ConditionalTransformError(
-            "'elif' branches are not supported when not using @qjit and the "
-            "conditional include mid-circuit measurements."
+            "'elif' branches are not supported when not using @qjit or with qml.capture.disabled()\n"
+            "if the conditional includes mid-circuit measurements."

```

Comment:
Either `qjit` or `capture.enabled` will make it work, right?
```suggestion
            "'elif' branches are not supported when not using @qjit and with qml.capture.disabled()\n"
            "if the conditional includes mid-circuit measurements."

```